Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift

```diff
@@ -71,8 +69,9 @@ public class RecentsListService: NSObject, RecentsListServiceProtocol {
         .home: [.invited, .favorited, .directHome, .conversationHome, .lowPriority, .serverNotice, .suggested],
         .favourites: [.favorited],
         .people: [.directPeople],
-        .rooms: [.conversationRooms, .suggested,
-            .invited, .favorited, .directHome, .directPeople, .conversationHome, .serverNotice]
+        .rooms: [.conversationRooms, .suggested]
```

Comment:
please add `.serverNotice` in this rooms list

Reply:
Done